Source/Blazorise/Components/Typography/OrderedList.razor.cs

```diff
@@ -64,5 +77,23 @@ public OrderedListType ListType
         }
     }
 
+    /// <summary>
+    /// Defines the marker images for list items.
+    /// </summary>
+    [Parameter]
+    public string ListStyleImage
```

Comment:
I almost argue that if this will stay as an URL. maybe the naming could the `ListStyleImageUrl`. But I'm not sure. What do you think?

At least mention in the summary that this is limited to **URL**.